src/platform/site-wide/user-nav/components/HelpMenu.jsx

```diff
@@ -22,17 +22,13 @@ class HelpMenu extends React.Component {
             <a href={`${FACILITY_LOCATOR_URL}`}>Find a VA Location</a>
           </p>
           <p>
-            <a href="https://iris.custhelp.va.gov/app/ask ">
-              Submit a Help Request
-            </a>
+            <b>Call MyVA311:</b>
           </p>
           <p>
-            <a href="https://iris.custhelp.va.gov ">Go to VA Help Center</a>
+            <a href="tel:18446982311">1-844-698-2311</a>
           </p>
           <p>
-            <a href="https://iris.custhelp.va.gov/app/answers/detail/a_id/1703 ">
-              Call Us{' '}
-            </a>
+            TTY: <a href="tel:711">711</a>
```

Comment:
No link necessary here - https://github.com/department-of-veterans-affairs/vets.gov-team/issues/13818#issuecomment-431510255.

Reply:
Great, Thanks. Will fix.